browser/brave_vpn/win/brave_vpn_wireguard_service/service/wireguard_tunnel_service.cc

```diff
@@ -293,7 +293,7 @@ bool CreateAndRunBraveWireguardService(const std::wstring& encoded_config) {
   // a desktop nitification popup with established connection information which
   // will be shown on each time.
   brave_vpn::RunTrayProcessAsUser();
-  return DeleteService(service.Get()) != 0;
```

Comment:
Just curious - previously service was deleted. Deleting service doesn't affect launched service?

Reply:
it marks launched service as deleted and the service will be deleted by SCM on stop automatically